code/tests/functional/tests/functions/advanced_image_processing/conftest.py

```diff
@@ -1,8 +1,14 @@
 import logging
+import os
+import sys
 import pytest
 from tests.functional.app_config import AppConfig
-from backend.batch.utilities.helpers.config.config_helper import ConfigHelper
-from backend.batch.utilities.helpers.env_helper import EnvHelper
+
+sys.path.append(
+    os.path.join(os.path.dirname(sys.path[0]), "..", "..", "..", "backend", "batch")
+)
+from utilities.helpers.config.config_helper import ConfigHelper  # noqa: E402
```

Comment:
Does this still complain in the IDE? I think it's probably worth adding a comment explaining that it's required due to the funky sys path stuff we have to do. Same for the int vec conftest.

Reply:
Added a comment, let me know if it conveys the message